### PR TITLE
Make Full Width Text organism a StreamBlock and add insets

### PR DESCRIPTION
**Pull request by kurtw**
- Add the insets Quote and Related Links.
- The existing Call to Action is also an inset.

@KimberlyMunoz
@kave

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/text-introduction.html [hunk 1]
```diff
@@ -23,15 +23,12 @@
...
     {% do global_dict.update(
         {
-            'introduction': {
+            'text_introduction': {
                 'heading':      'Text Introduction',
                 'intro':        'Content Intro Lorem Ipsum',
-                'body': {
-                    'source': lipsumText
-                },
+                'body':         lipsum(1, false, 10, 25) | safe,
```
Comment (kave) [inline cfgov/jinja2/v1/_includes/molecules/text-introduction.html]: inner child source got overritten

**Comment by kurtw** [comment]
Ready for review.

File: cfgov/jinja2/v1/_includes/molecules/quote.html [hunk 2]
```diff
@@ -8,21 +8,26 @@
...
-   settings.citation:   String with the author or citation of text.
+   value.citation:   String with the author or citation of text.
 
    ========================================================================== #}
 
-{% macro render(settings) %}
+{% if not value %}
+    {% do global_dict.update({
+      'quote': {
+        'quote': 'This is a quote.',
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/molecules/quote.html]: Can we rename this `body` so the final object is `quote.body` and it matches the markup?

File: cfgov/jinja2/v1/_includes/organisms/full-width-text.html [hunk 3]
```diff
@@ -10,20 +10,84 @@
...
-            {% import template as content %}
-                {{ content }}
-        </div>
-    {% endmacro %}
+{% if not value %}
+    {% do global_dict.update({
+        'full_width_text': [
+            {
+                'block_type': 'content',
+                'source': """
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/organisms/full-width-text.html]: Whoa, you can do triple quotes for a large block?

Comment (kurtw) [inline cfgov/jinja2/v1/_includes/organisms/full-width-text.html]: It's python baby.

**Comment by KimberlyMunoz** [comment]
There's an error with the table macro that breaks `/learn-page/` when I test. If rebase with flapjack, I've removed it from that page to keep testing. Code looks great, I just want to visually check the spacing before signing off.

**Comment by kurtw** [comment]
Ready for another review!

**Comment by KimberlyMunoz** [comment]
:+1:

**Pull request merged**